control-plane/connect-inject/webhook/consul_dataplane_sidecar.go

```diff
@@ -97,9 +97,10 @@ func (w *MeshWebhook) consulDataplaneSidecar(namespace corev1.Namespace, pod cor
 	}
 
 	container := corev1.Container{
-		Name:      containerName,
-		Image:     w.ImageConsulDataplane,
-		Resources: resources,
+		Name:            containerName,
+		Image:           w.ImageConsulDataplane,
+		ImagePullPolicy: corev1.PullPolicy(w.GlobalImagePullPolicy),
```

Comment:
is the idea that there's a global imagePullPolicy and that each stanza should be able to overwrite that with their own pull policy?

Reply:
Good question, I think the idea was to only have a global pull policy. I could see the usefulness of overwrite, but I don't think we have any plans to do that right now.